compiler/rustc_pattern_analysis/src/pat.rs

```diff
@@ -10,6 +10,7 @@ use crate::usefulness::PlaceCtxt;
 use crate::{Captures, TypeCx};
 
 use self::Constructor::*;
+use self::PatOrWild::*;
```

Comment:
I actually find it harder to read with the glob import, since now we have `PatOrWild::Wild` and `Constructor::Wildcard` in scope. Is it really worth the glob import?

Reply:
Nah you're right, it's clearer without